ui_framework/src/components/header/_index.scss

```diff
@@ -0,0 +1,26 @@
+// Themable colors
+$kuiHeaderBackgroundColor: $kuiColorWhite;
+$kuiHeaderBreadcrumbColor: $kuiColorDarkestGray;
+$kuiHeaderNotificationBackgroundColor: $kuiColorAccent;
+
+// Layout vars
+$kuiHeaderChildSize: $kuiSizeXXL + $kuiSizeL;
+
+
+// Mixin to add a border to the left or right of an item. Requires relative position on the parent.
+@mixin kuiHeaderSeparation($position) {
```

Comment:
Can we use the term "header item" in the name of this mixin?

Also, if we were to discuss the design of the item, what noun would we use to refer to the vertical line? A "divider" or maybe a "separator"? Can we change this mixin to use that term, e.g. `kuiHeaderItemSeparator` or `kuiHeaderItemDivider`?